.github/workflows/build_patch_artifacts.yaml

```diff
@@ -0,0 +1,31 @@
+name: build_patch_artifacts
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
not entirely sure what this does, but also not sure if you want this to run on pull requests (at least the upload portion).

Reply:
I'm just commenting out to test and will re-add this when I've confirmed it works.